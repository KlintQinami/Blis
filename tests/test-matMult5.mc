int main() {
  vec3 x1 = vec3(1., 2., 3.);
  mat3x1 A = mat3x1(4., 5., 6.);      
  mat3x3 P = x1 * A;
  A = A * P;
  printf(A.x);
  printf(A.y);
  printf(A.z);
  return 0;
}
