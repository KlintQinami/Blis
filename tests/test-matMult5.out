128.000000
160.000000
192.000000
